src/pages/signin/ChooseSSOOrMagicCode.js

```diff
@@ -79,21 +70,12 @@ function ChooseSSOOrMagicCode({credentials, account, setIsUsingMagicCode}) {
                     </Text>
                 </View>
 
-                <Button
-                    isDisabled={isOffline}
-                    style={[styles.mv3]}
-                    text={translate('samlSignIn.useMagicCode')}
-                    isLoading={account.isLoading && account.loadingForm === (account.requiresTwoFactorAuth ? CONST.FORMS.VALIDATE_TFA_CODE_FORM : CONST.FORMS.VALIDATE_CODE_FORM)}
-                    onPress={() => {
-                        Session.resendValidateCode(credentials.login);
-                        setIsUsingMagicCode(true);
-                    }}
+                <ValidateCodeForm
+                    setIsUsingMagicCode={setIsUsingMagicCode}
+                    isUsingRecoveryCode={false}
+                    setIsUsingRecoveryCode={() => undefined}
```

Comment:
Why don't we maintain the possibility of using recovery codes for the users?

Reply:
Good question, I thought this was outside the scope of my proposal., but I made the change, and now the user going to be able to use the recovery code option in this component.

Reply:
We need to end up with correctlwy working sing in flow so it's required